paper-tooltip.html

```diff
@@ -364,12 +345,51 @@
 
       },
 
+      _addListeners: function() {
+        if (this._target) {
+          this.listen(this._target, 'mouseenter', 'show');
+          this.listen(this._target, 'focus', 'show');
+          this.listen(this._target, 'mouseleave', 'hide');
+          this.listen(this._target, 'blur', 'hide');
+          this.listen(this._target, 'tap', 'hide');
+        }
+        this.listen(this, 'mouseenter', 'hide');
+      },
+
+      _findTarget: function() {
+        if (!this.manualMode)
+          this._removeListeners();
```

Comment:
Blerg, this blows that it's being called on `attached`, where you don't have any listeners to remove. But I guess it won't do anything because `this._target` is null?